### PR TITLE
kick off production users list with Apester, MayaData and Weaveworks

### PR DESCRIPTION
**Pull request by dholbach**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,11 @@ Interact with your containers directly: pause, restart, and stop containers. Lau
 
 Add custom details or interactions for your hosts, containers, and/or processes by creating Scope plugins. Or, just choose from some that others have already written at the GitHub [Weaveworks Scope Plugins](https://github.com/weaveworks-plugins/) organization.
 
+## Who is using Flux in production
```
Comment (qiell) [inline README.md]: I think it should be `Scope` instead of `Flux`.

Comment (dholbach) [reply README.md]: Fixed.

File: README.md [hunk 2]
```diff
@@ -32,6 +32,11 @@ Interact with your containers directly: pause, restart, and stop containers. Lau
 
 Add custom details or interactions for your hosts, containers, and/or processes by creating Scope plugins. Or, just choose from some that others have already written at the GitHub [Weaveworks Scope Plugins](https://github.com/weaveworks-plugins/) organization.
 
+## Who is using Scope in production
+
+- [MayaData](https://mayadata.io/) in [MayaOnline](https://mayadata.io/products)
```
Comment (muratkars) [inline README.md]:
```suggestion
- [MayaData](https://mayadata.io/) in [MayaOnline / MayaOnPrem](https://mayadata.io/products)
```

**Comment by vic3lord** [comment]
Hi @dholbach, we at Apester use scope in production https://apester.com

**Comment by dholbach** [comment]
Let's land this for now and then do consecutive PRs for new users of Scope.

**Pull request merged**